charts/newrelic-infrastructure/templates/_helpers.tpl

```diff
@@ -35,6 +35,15 @@ chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
 heritage: {{.Release.Service }}
 release: {{.Release.Name }}
 mode: {{ template "newrelic.mode" . }}
+{{- if .Values.global }}
+  {{- if .Values.global.customLabels }}
+{{ toYaml .Values.global.customLabels }}
+  {{- end }}
+{{- else }}
+  {{- if .Values.customLabels }}
```

Comment:
I would expect local configuration take precedence over global ones, but I see this behavior in the rest of globals here. @roobre  wdyt?

Reply:
Could you change this logic so both labels are added, meaning global labels will be added to all charts and local labels will be added just in this chart resources.